deploy/docs/fluentd_otc_comparison.md

```diff
@@ -0,0 +1,212 @@
+| `verify_ssl`                                               | No direct translation. Please use [processors.k8s_tagger.auth_type: kubeconfig][kubeconfig_auth_type] |
+| `client_cert`                                              | No direct translation. Please use [processors.k8s_tagger.auth_type: kubeconfig][kubeconfig_auth_type] |
+| `client_key`                                               | No direct translation. Please use [processors.k8s_tagger.auth_type: kubeconfig][kubeconfig_auth_type] |
+| `bearer_token_file`                                        | No direct translation. Please use [processors.k8s_tagger.auth_type: kubeconfig][kubeconfig_auth_type] |
+| `cache_size`                                               | N/A                                                                                                   |
+| `cache_ttl`                                                | N/A                                                                                                   |
+
+[fluent_plugin_k8s_metadata]: https://github.com/SumoLogic/sumologic-kubernetes-fluentd/tree/v1.12.2-sumo-4/fluent-plugin-kubernetes-metadata-filter#configuration
+[k8sprocessor]: https://github.com/SumoLogic/sumologic-otel-collector/blob/v0.0.28-beta.0/pkg/processor/k8sprocessor
+[k8sprocessor_field_extract]: https://github.com/SumoLogic/sumologic-otel-collector/tree/v0.0.28-beta.0/pkg/processor/k8sprocessor#field-extract-config
+[kubeconfig_auth_type]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.36.0/internal/k8sconfig/config.go#L53-L60
+
+### fluent-plugin-enhance-k8s-metadata
+
+| [Fluentd configuration option][fluent_plugin_enhance_k8s_metadata] | [Opentelemetry Kubernetes Processor][k8sprocessor]                                      |
+|--------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| `cache_size`                                                       | N/A                                                                                     |
+| `cache_ttl`                                                        | N/A                                                                                     |
+| `cache_refresh`                                                    | N/A                                                                                     |
+| `cache_refresh_variation`                                          | N/A                                                                                     |
+| `in_namespace_path`                                                | Uses [processors.k8s_tagger.pod_association: [{from: build_hostname}]][pod_association] |
+| `in_pod_path`                                                      | Uses [processors.k8s_tagger.pod_association: [{from: build_hostname}]][pod_association] |
```

Comment:
One nit (perhaps this is not possible): can markdown table cells include multiline code? (Maybe it couldn't, can't recall)

This way it would be easier to read but if it's not possible then 🤷

Reply:
it cannot :disappointed: 
https://stackoverflow.com/questions/15955160/markdown-table-with-long-lines

unless we use HTML :vomiting_face: